addon-test-support/wait-until.js

```diff
@@ -5,7 +5,7 @@ export function waitUntil(callback, { timeout = 1000 } = {}) {
   return new RSVP.Promise(function(resolve, reject) {
     let value = run(callback);
     if (value) {
-      resolve(value);
+      return resolve(value);
```

Comment:
Sure. Thought about this myself, but also believe the return value has no meaning. But code maybe looks misleading as if it does have a meaning what we return here...